EU4toV2/Source/Mappers/CountryFlags/CountryFlags.h

```diff
@@ -0,0 +1,23 @@
+#ifndef COUNTRY_FLAGS_H
+#define COUNTRY_FLAGS_H
+
+#include "Parser.h"
+
+namespace mappers
+{
+	class CountryFlags: commonItems::parser
+	{
+	public:
+		CountryFlags();
+
+		void registerKeys();
+
+		[[nodiscard]] const auto& getFlags() const { return flags; }
+	
+	private:
+		std::map<std::string, std::vector<std::string>> flags;
```

Comment:
The way this is used, you might instead use a std::multimap - https://en.cppreference.com/w/cpp/container/multimap

If you do, equal_range() is how you extract the whole set of matching items.

Reply:
Done